src/components/SignIn.tsx

```diff
@@ -69,6 +73,9 @@ const SignIn: FC = () => {
       setShouldValidate(true);
     } else {
       signIn({ email: lowercaseEmail });
+      if (signInSuccess) {
+        setSuccessView(true);
+      }
```

Comment:
Probably this doesn't work.
I think you need to change `signIn` to `mutateAsync` so you can await it. Then you can know if the request worked or not, and set the success view accordingly.

Reply:
The problem why it probably does not work is that `isSuccess` comes from a hook. In the case `isSuccess` changes, it will re-render the component, but not run at line 76 (that is an event trigger, not run during component rendering).